frontend/src/modules/alarms/AlarmsPage.tsx

```diff
@@ -478,17 +469,14 @@ export const AlarmsPage = (): JSX.Element => {
                     </Grid>
                     <Grid item className={classes.marginContent}>
                       {alarmConfig.map((param) => {
-                        if (
-                          alarmLimits[param.stateKey].lower !==
-                            alarmLimitsDraft[param.stateKey]?.lower ||
-                          alarmLimits[param.stateKey].upper !==
-                            alarmLimitsDraft[param.stateKey]?.upper
-                        ) {
+                        if (alarmLimitsUnsavedKeys.includes(param.stateKey)) {
                           return (
                             <Typography variant="subtitle1">{`Keep ${param.label} alarm range to ${
-                              alarmLimits[param.stateKey].lower
+                              alarmLimits !== null && alarmLimits[param.stateKey].lower
                             } -
-                                ${alarmLimits[param.stateKey].upper}?`}</Typography>
+                                ${
+                                  alarmLimits !== null && alarmLimits[param.stateKey].upper
+                                }?`}</Typography>
```

Comment:
Shouldn't these be using `getAlarmLimitsRequest` rather than `getAlarmLimitsCurrent`?